internal/blueprint/blueprint_test.go

```diff
@@ -3,6 +3,7 @@ package blueprint
 import (
 	"github.com/google/go-cmp/cmp"
 	"testing"
+    "github.com/stretchr/testify/assert"
```

Comment:
Looks like tab vs spaces issue. Try `./tools/prepare-source.sh`

Reply:
Thanks, that helped. I also added all the things it generated like @ondrejbudai once told me to do. Should also fix the failing CI.